### PR TITLE
Updated package metadata to reflect new version

### PR DESCRIPTION
**Pull request by sonofmun**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 dist: xenial
 language: python
 python:
-  - "3.5"
```
Comment (rillian) [inline .travis.yml]: I would be helpful to mention in the body of the commit message that Python 3.5 is now out of support.

Could add in 3.8 as well, but that's not important for this PR.

**Review by rillian — APPROVED** [review]
LGTM FWIW.

File: .travis.yml (hunk 1 above)
Comment (sonofmun) [reply .travis.yml]: Thanks for the tip about adding removal of 3.5 support. I have added that note both to the CHANGES.txt file and to the commit message there.

**Comment by sonofmun** [comment]
Once @PonteIneptique has reviewed and approved, I will go ahead and process the new release.

**Review by PonteIneptique — APPROVED** [review]
Agrees with @rillian about 3.8
But another PR :)

**Comment by sonofmun** [comment]
I will wait to add 3.8 to the next PR.

**Comment by rillian** [comment]
Thanks!

**Pull request merged**